### PR TITLE
From Feature/profile to Master

### PR DESCRIPTION
**Pull request by halmasieh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,87 @@
...
...kills in Excel, VBA, Python, R, JavaScript, SQL Databases, Tableau, Big Data, and Machine Learning. I worked with various datasets during the BootCamp program and according to the content of the projects, I used various tools to analyze the data, design the model and visualize the finding through Tableau and dashboards. In addition, I am a former Assistant Professor of University in the area of Applied Mathematics with a brilliant and valuable background in the field of teaching and scientific research. With this extensive background of Mathematics and Data, I started  my training program at Jarvis by developing various projects and this mentorship helped me to gain skills in increasing code efficiency, debugging, and gaining comprehensive knowledge in Linux/Bash, Git, Java, Docker, RDBMS and Springboot. I am a self-initiated and avid learner with high attention to detail, critical thinking and logical problem-solving. My adaptability allows me to work in many different environments, in addition to speaking with people from various backgrounds. I am looking forward to speaking about my academic and professional background for developing positions mainly in the data analytics field.
```
Comment (jarviscanada) [inline README.md:3]: I recently completed Data Analytics BootCamp from the University of Toronto focused on gaining technical programming skills in Excel, VBA, Python, R, JavaScript, SQL Databases, Tableau, Big Data, and Machine Learning. I used various tools to analyze the data, design the model and visualize the finding through Tableau and dashboards. In addition, I am a former Assistant Professor of University in Applied Mathematics with a brilliant and valuable background in teaching and scientific research. With this extensive mathematics and data background, I started my training program at Jarvis by developing various projects. This mentorship helped me increase code efficiency, debugging, and comprehensive knowledge in Linux/Bash, Git, and Java Docker, RDBMS and Springboot. I am a self-initiated and avid learner with great attention to detail, critical thinking and logical problem-solving. My adaptability allows me to work in many different environments and speak with people from various backgrounds. I am looking forward to discussing my academic and professional experience in developing positions in the data analytics and software fields.
		
did some minor fixes

File: README.md [hunk 2]
```diff
@@ -1,8 +1,87 @@
...
+**Competent:** Pandas, Numpy, Scipy, Scikit-Learn, DHTML/CSS
+
+**Familiar:** R, APIs, Tensorflow/Deep learning, Flask, Google Cloud Platform(GCP)
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/linux_sql)]: Implemented a Monitoring Agent on a node/server within the linux cluster to collect the hardware specifications and monitor the cluster node resource usage in real time. This Monitoring Agent contains a PostgreSQL server installed within a Docker container that collects information submitted through bash scripts installed on each node.
```
Comment (jarviscanada) [inline README.md:18]: Implemented a Monitoring Agent on a node/server within the Linux cluster to collect the hardware specifications and monitor the cluster node resource usage in real-time. This Monitoring Agent contains a PostgreSQL server installed within a Docker container that collects information submitted through bash scripts installed on each node.

fixed typo

Comment (jarviscanada) [reply README.md:18]: missing keywords
https://github.com/jarviscanada/jarvis_profile_builder/blob/master/profile.yaml
image

File: README.md [hunk 3]
```diff
@@ -1,8 +1,87 @@
...
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/cloud_devops)]: Not Started
```
Comment (jarviscanada) [inline README.md:34]: remove not started projects

**Review by jarviscanada — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (halmasieh) [reply README.md:18]: I have added the keywords to the project description.

**Pull request merged**